padd.sh

```diff
@@ -147,7 +133,7 @@ GetFTLData() {
 
 GetSummaryInformation() {
   local summary
-  local cache_summary
+  #local cache_summary
```

Comment:
Don't comment out the code if you don't want it. You can delete the lines, this is `git` so the previous version is there in the history and `git blame` if we need to revert anything.